src/user/manage-users-user-roles-and-user-groups.md

```diff
@@ -193,6 +193,20 @@ want to, you may also enter the user name that the account will have. If
 you leave the user name empty, then the user may choose their own user
 name when they respond to the invitation (as long as it is not taken
 already for another user.)<br/><br/>
+
+#### Username requirements
```

Comment:
Could you please move this section below "Create account with user details" instead of the current positioning below the invite by email?

Reply:
Hi @Topener Changes made, please review.